src/applications/vaos/tests/new-appointment/components/TypeOfFacilityPage.unit.spec.jsx

```diff
@@ -24,11 +25,16 @@ const initialState = {
 };
 
 describe('VAOS integration: VA facility page with a single-site user', () => {
-  beforeEach(() => mockFetch());
+  let store;
+
+  beforeEach(() => {
+    mockFetch();
+    store = createTestStore(initialState);
+    store.dispatch(startDirectScheduleFlow());
```

Comment:
What I meant in that comment was that you can write a test in the reason for appointment page test that dispatches this action, renders the reason for appointment page, and verifies that the character limit for the reason box changes depending on which radio option you choose. You shouldn't need to modify this test.

Reply:
Resolved, the naming of the test could probably be better but I believe this is the test you are seeking.